app/assets/v2/scss/gitcoin.scss

```diff
@@ -203,6 +205,10 @@
   background-color: #F5F2FF;
 }
 
+.bg-violet-300 {
+  background-color: $gc-violet-300;
+}
+
```

Comment:
Consider: Should we move all of these color/font utilities from `gitcoin.scss` to `gc-utilities.scss`?